common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -230,6 +230,29 @@ public static Serializable base64StringToSerializable(String input) throws IOExc
         return answer;
     }
 
+    public static Map<String, Serializable> mapOfBase64StringToSerializable(Map<String, String> input) throws IOException, ClassNotFoundException {
+        if (input == null) {
+            return null;
+        }
+        HashMap<String, Serializable> answer = new HashMap<>(input.size());
+        for (Map.Entry<String, String> entry : input.entrySet()) {
+            answer.put(entry.getKey(), base64StringToSerializable(entry.getValue()));
+        }
+        return answer;
+    }
+
+    public static Map<String, String> mapOfSerializableToString(Map<String, Serializable> input)
+            throws IOException, ClassNotFoundException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.ClassNotFoundException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)